server/cluster/cluster.go

```diff
@@ -422,7 +419,7 @@ func (c *RaftCluster) checkTSOService() {
 				log.Info("TSO is provided by PD")
 				c.UnsetServiceIndependent(constant.TSOServiceName)
 			} else {
-				if err := c.startTSOJobsIfNeeded(); err != nil {
+				if err := c.stopTSOJobsIfNeeded(); err != nil {
```

Comment:
Is the previous startTSOJobsIfNeeded a mistake?

Reply:
Yes. (